src/audiovideocontroller/AudioVideoControllerState.ts

```diff
@@ -110,8 +110,6 @@ export default class AudioVideoControllerState {
 
   videoInputAttachedTimestampMs: number = 0;
 
-  audioDeviceInformation: { [id: string]: string } = {};
```

Comment:
we do publish video device information through event controller.  I am more curious about how to trigger the track label undefined condition ? Any idea ?

Reply:
This was due to a bug filed here: https://github.com/aws/amazon-chime-sdk-js/issues/1103. Looks like the device label is undefined